README.md

```diff
@@ -10,3 +10,10 @@ Sample `.env` file
 API_URL=http://rickandmortyapi.com/api
 PRODUCTION=false
 ```
+
+###Konwencja commitów
+
+1. angielski
+2. czas przeszły
+3. zgodnie ze specyfikacją commitów na Conventional Commits
+4. znaczniki + kod tasku + treść
```

Comment:
fajnie, że dodajemy readmy, dorzuciłbym przykładowy poprawny commit

Reply:
Skonsultuje z grupa czy aby na pewno dobrze pisze commity tak jak to zakładaliśmy, wydaje mi się, ze nie spełniam warunku 4 w pełni poprawnie